code/modules/clothing/gloves/color.dm

```diff
@@ -12,6 +12,52 @@
 	custom_price = 1200
 	custom_premium_price = 1200
 
+/obj/item/toy/sprayoncan
+	name = "spray-on insuls applicator"
+	desc = "What is the number one problem facing our station today?"
+	icon = 'icons/obj/clothing/gloves.dmi'
+	icon_state = "sprayoncan"
+
+/obj/item/toy/sprayoncan/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(iscarbon(target) && proximity)
+		var/mob/living/carbon/C = target
+		var/mob/living/carbon/U = user
+		var/success = C.equip_to_slot_if_possible(new /obj/item/clothing/gloves/color/yellow/sprayon, ITEM_SLOT_GLOVES, TRUE, TRUE)
+		if(success)
+			if(C == user)
+				C.visible_message("<span class='notice'>[U] sprays their hands with glittery rubber!</span>")
+			else
+				C.visible_message("<span class='warning'>[U] sprays glittery rubber on the hands of [C]!</span>")
+		else
+			C.visible_message("<span class='warning'>The rubber fails to stick to [C]'s hands!")
+		qdel(src)
+
+/obj/item/clothing/gloves/color/yellow/sprayon
+	desc = "How're you gonna get 'em off, nerd?"
+	name = "spray-on insulated gloves"
+	icon_state = "sprayon"
+	item_state = "sprayon"
+	permeability_coefficient = 0
+	resistance_flags = ACID_PROOF
+	var/shocks_remaining = 10
+
+/obj/item/clothing/gloves/color/yellow/sprayon/Initialize()
+	.=..()
+	ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT) //Cursed with the power of science, or something

```

Comment:
It's not cursed item, change the source to something more appriopriate.

Reply:
To be fair it's pretty cursed but not in that way